app/javascript/lib/scp-api.js

```diff
@@ -244,22 +244,25 @@ export async function fetchClusterOptions(studyAccession, mock=false) {
   return values
 }
 
+
 /**
  * Returns an object with scatter plot data for a cluster in a study
  *
  * See definition: app/controllers/api/v1/visualization/clusters_controller.rb
  *
  * @param {String} studyAccession Study accession
  * @param {String} cluster Name of cluster, as defined at upload
- * @param {String} annotation Full annotation name, e.g. "CLUSTER--group--study", or object with name,type, and scope properties
+ * @param {String} annotation Full annotation name,
+     e.g. "CLUSTER--group--study", or object with name,type, and scope properties
  * @param {String} subsample Subsampling threshold, e.g. 100000
  * @param {String} consensus Statistic to use for consensus, e.g. "mean"
  * @param {Boolean} isAnnotatedScatter If showing "Annotated scatter" plot.
  *                  Only applies for numeric (not group) annotations.
  * @param {Boolean} mock If using mock data.  Helps development, tests.
  *
  * Example:
- * https://localhost:3000/single_cell/api/v1/studies/SCP56/clusters/Coordinates_Major_cell_types.txt?annotation_name=CLUSTER&annotation_type=group&annotation_scope=study
+ * https://localhost:3000/single_cell/api/v1/studies/SCP56/clusters/
+     Coordinates_Major_cell_types.txt?annotation_name=CLUSTER&annotation_type=group&annotation_scope=study
```

Comment:
ESLint also substantially degrades readability here, so let's pass over this case.

```suggestion
 * // eslint-disable-next-line
 * https://localhost:3000/single_cell/api/v1/studies/SCP56/clusters/Coordinates_Major_cell_types.txt?annotation_name=CLUSTER&annotation_type=group&annotation_scope=study
```

Reply:
Thoughts on this, @devonbush?